src/App.tsx

```diff
@@ -1,9 +1,68 @@
 import { useState } from 'react';
-import { Button } from 'react-bootstrap';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Image from 'react-bootstrap/Image';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
 import './App.scss';
-
+import Categories from './components/Categories.jsx';
+import Event from './components/Event.jsx';
+import Header from './components/Header.jsx';
```

Comment:
You components should have tsx extension and you don't have to include extension here